environment.yaml

```diff
@@ -0,0 +1,16 @@
+name: openfold-env
+channels:
+- conda-forge
+- bioconda
+- defaults
```

Comment:
Note: as soon as you have defaults here commercial entities can no longer use this script which isn't great for those of us in commercial entities.